pages/admin/users/index.tsx

```diff
@@ -1,8 +1,280 @@
-import { AdminLayout } from '@lib/components/Admin';
+import { AdminLayout, AdminTablePagination } from '@lib/components/Admin';
 import type { NextPage } from 'next';
 import Head from 'next/head';
+import {
+  Button,
+  DebouncedInput,
+  IndeterminateCheckbox,
+} from '@components/Input';
+import React, { useMemo, useState } from 'react';
+import { CircleCheck, CircleXMark, Search } from '@lib/icons';
+import {
+  ColumnFiltersState,
+  createColumnHelper,
+  getCoreRowModel,
+  getFilteredRowModel,
+  getPaginationRowModel,
+  getSortedRowModel,
+  SortingState,
+  useReactTable,
+} from '@tanstack/react-table';
+import Link from 'next/link';
+import { AdminTable } from '@components/Admin';
+import { getLocaleDateString, getMembershipYear } from '@lib/utils';
 
+type AdminUserType = {
```

Comment:
Lagde ny type siden UserType har for mange types som vi ikke skal hente her i admin. Bedre måte å gjøre dette på?

Reply:
Du kan egt. bare hente User typen fra prisma i dette tilfellet